docs/03-apis/api-v2/editing-values.md

```diff
@@ -431,7 +436,10 @@ the resource to the value, and the value's ID and type. For example:
 ```
 
 The optional property `knora-api:deleteComment` specifies a comment to be attached to the
-value, explaining why it has been marked as deleted.
+value, explaining why it has been marked as deleted
+
+The optional property `knora-api:deleteDate` (an [xsd:dateTimeStamp](https://www.w3.org/TR/xmlschema11-2/#dateTimeStamp))
+specifies a custom timestamp indicating when the value was deleted. If not specified, the current time is used.å
```

Comment:
what does the Swedish character å doing at the end of this line? :-D I actually like it giving knora a scandinavian flavor. Did you know this single character means a river?

Reply:
I often find those "rivers" in my code, I think it happens when I accidentally press Option-A instead of Command-S. 😀